.travis.yml

```diff
@@ -14,7 +13,7 @@ services:
   - mysql
 
 git:
-  depth: 1
+  depth: 10
```

Comment:
why this change?

Reply:
The travis.yml was copied from another module, i don't understand that change either but i keeped it like on other modules

Reply:
not sure what it does, probably for some recursive stuff

Reply:
Travis is now okay, tell me if i should mark this as resolved

Reply:
yes its ok

Reply:
in fact it's the number of commits when you clone @wizzymore  (so latest commit + 10), dunno if we need 10, 1 or 0. 10 is fine tho, but just saying